src/cli/lint.test.ts

```diff
@@ -200,10 +202,67 @@ describe('TypeScript', () => {
         getArea(): number;
       }
 
-      // const Ctor: abstract new () => HasArea = Shape;
-      const Ctor: new () => HasArea = Shape;
+      const Ctor: abstract new () => HasArea = Square;
 
       expect(Ctor).toBeDefined();
     });
   });
+
+  describe('4.3', () => {
+    test.skip('Separate Write Types of Properties', () => {
```

Comment:
Not seeing much success in ESLint/Prettier picking up these new syntaxes. Will have to dig a little further.